### PR TITLE
Fix and complete the secure connection implementation

### PR DESCRIPTION
**Pull request by Drapersniper**

### CODE REVIEW

File: lavalink/node.py [hunk 1]
```diff
@@ -225,8 +227,9 @@ async def connect(self, timeout=None, secured=False):
         asyncio.TimeoutError
             If the websocket failed to connect after the given time.
         """
+        self.secured = secured
```
Comment (Jackenmen) [inline lavalink/node.py]: I feel like we just shouldn't have the `secured` argument in this method. Especially that it kinda defeats the point of having the `secured` attribute if it always gets overridden on connect.

Comment (Drapersniper) [reply lavalink/node.py]: Its here to not break 0.9.2 - if removed this is a breaking change.

But yes I agree with you.

Comment (Jackenmen) [reply lavalink/node.py]: > Its here to not break 0.9.2 - if removed this is a breaking change.

I'm fine with breaking here, you can probably count affected people on one hand :)

Comment (Drapersniper) [reply lavalink/node.py]: Done

**Review by Jackenmen — APPROVED** [review]
LGTM

but does it look good to you?

**Pull request merged**